examples/simpleReceive.ts

```diff
@@ -0,0 +1,51 @@
+import { OnMessage, OnError, MessagingError, delay, Message, ReceiveMode, Namespace } from "../lib";
+// import * as rhea from "rhea";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = "SERVICEBUS_CONNECTION_STRING";
+const entityPath = "QUEUE_NAME";
+const str = process.env[connectionString] || "";
+const path = process.env[entityPath] || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+let ns: Namespace;
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createQueueClient(path, { receiveMode: ReceiveMode.peekLock, maxConcurrentCalls: 1 });
```

Comment:
i know we talked about this, but can you remind me why the receive mode and concurrency are on the client rather than when you call `client.receive()`? they don't affect the sender at all so it seems odd for them to be at the client level

Reply:
I feel the same. 2 reasons for putting it at client level
- .net sdk has it at the client level.
- Wanted receiveBatch() and receive() both to have same modes. Moreover once receive() is called, it cannot be called another time without calling receiveHandler.stop(). Hence thought of having them as client level options.

Reply:
i see. i can see it for receiveMode, but the only place where the maxConcurrentCalls has any meaning is on `receive()`. the batching receiver manages the credits in a different way so i think we should move `maxConcurrentCalls` down to the `receive()` options